CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Breaking change
 
+* Mechanism for resubmitting tasks was changed. Command `resubmit` was removed, see the manual for replacement.
```

Comment:
We could add a link here that points to the new resubmit mechanism. "see the manual" is quite generic :)